pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/SimplePlayerActivity.kt

```diff
@@ -108,7 +109,10 @@ class SimplePlayerActivity : ComponentActivity(), ServiceConnection {
     }
 
     private fun isPictureInPicturePossible(): Boolean {
-        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.N
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+            return packageManager.hasSystemFeature(PackageManager.FEATURE_PICTURE_IN_PICTURE)
+        }
+        return false
```

Comment:
Should the same change also be done here?
https://github.com/SRGSSR/pillarbox-android/blob/8e65327202f5a57cd77f47996eccea74f022d564/pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/showcases/integrations/MediaControllerActivity.kt#L71

Reply:
I've added the additional check in `MediaControllerActivity` too in https://github.com/SRGSSR/pillarbox-android/pull/663/commits/e0ce8e46671f90d0da583d7cb2debb26381447d1